### PR TITLE
Setup travisci

### LINKED ISSUE #7: Add initial and simple Github Action to have pytest run on new pull requests
Initially, Github actions for this project could be very simple and scaled to handle the more complex tasks as needed. In my point of view, the first task where automation is needed is the run of pytest on new pull requests. So then we can have a sanity check on the new code.

### PR DESCRIPTION
**Pull request by LeonardoFurtado**
travis.yml file is necessary to setup travis ci and coveralls.

This closes #7

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+python:
+ - "3.6"
+
+sudo: required
+
+install:
+ - pip install -r requirements.txt
+ - pip install -e .
+ - pip install pytest
```
Comment (marceloFA) [inline .travis.yml:11]: Could you add these test-related dependencies in a file called dev-requirements.txt? So we can have a concise list of development requirements. If this suggestion is implemented, lines 11, 12, and 13 would be replaced by ``- pip3 install -r dev-requirements.txt ``

**Review by marceloFA — CHANGES_REQUESTED** [review]
I left a review for you. Thanks for your nice work here!

File: .travis.yml (hunk 1 above)
Comment (marceloFA) [reply .travis.yml:11]: also for this, we'd need to remove ``pytest``,``httmock``, and ``pylint`` from the standard requirements file. These packages would now need to be inside ``dev-requirements.txt``

**Comment by marceloFA** [comment]
Also, can we change from Travis to Github Actions? @LeonardoFurtado 
Here are my reasons for using Actions:
- It's already integrated into the platform so we don't have to do third-party integrations for now;
- No credentials handling and dependencies needed;
- For the simplicity of the project, Actions, as simples as it is, serves us quite nicely;
- the pipeline setup is easier (we're beginners);

File: .travis.yml (hunk 1 above)
Comment (LeonardoFurtado) [reply .travis.yml]: Done ;)

**Comment by LeonardoFurtado** [comment]
> 
> 
> Also, can we change from Travis to Github Actions? @LeonardoFurtado
> Here are my reasons for using Actions:
> 
>     * It's already integrated into the platform so we don't have to do third-party integrations for now;
> 
>     * No credentials handling and dependencies needed;
> 
>     * For the simplicity of the project, Actions, as simples as it is, serves us quite nicely;
> 
>     * the pipeline setup is easier (we're beginners);

I have some years experience with travisCI but I don't have any experience with Actions. But I feel comfortable to learn more about Actions

**Pull request closed without merging**